### PR TITLE
es6 changes in non-test code

### PR DESCRIPTION
**Pull request by nikulkarni**
Inspired by https://github.com/paypal/nemo/pull/159

Will submit a new PR for updating test code. Ran all unit tests and they all passed.

image

### CODE REVIEW

**Comment by skratchdot** [comment]
Seems like CI is failing due to jshint errors. LGTM besides that.

**Comment by skratchdot** [comment]
Seems like we should share lint configs, so change:
https://github.com/paypal/nemo-view/blob/master/.jshintrc
to match:
https://github.com/paypal/nemo/blob/master/.jshintrc

But I'm also not sure the best way to synchronize everything.

**Comment by nikulkarni** [comment]
For now I have fixed it by adding just the `esversion` in the jshint config. Now all is well

image

**Comment by skratchdot** [comment]
This looks good to me (but I'll give more time for review- or let someone else click merge) :)

**Comment by nikulkarni** [comment]
Sounds good, thanks @skratchdot

File: lib/locreator.js [hunk 1]
```diff
@@ -1,123 +1,100 @@
...
-  };
-};
+const Drivex = require('selenium-drivex');
+const Locatex = require('./locatex');
+const _ = require('lodash');
+const normalize = require('./normalize');
+const debug = require('debug');
+const log = debug('nemo-view:log');
 
+class Locreator {
```
Comment (grawk) [inline lib/locreator.js]: finally nemo gets some Class!!

**Pull request merged**